internal/handlers/common/getmore.go

```diff
@@ -38,33 +41,83 @@ func GetMore(ctx context.Context, msg *wire.OpMsg, registry *cursor.Registry) (*
 		return nil, err
 	}
 
-	collection, err := GetRequiredParam[string](document, "collection")
+	// TODO: Use ExtractParam https://github.com/FerretDB/FerretDB/issues/2859
+	v, err := document.Get("collection")
 	if err != nil {
```

Comment:
```suggestion
	v, _ := document.Get("collection")
	if v == nil {
```

Reply:
Will apply this in another PR too :pray: